### PR TITLE
add smurffy's baltimore file

### LINKED ISSUE #24: smurffy's workflow for adding Baltimore to the repo
## Baltimore, MD
- [x] Create an Issue
- [x] Create a branch
- [x] Make changes
- [ ] Open a PR (starts discussion)
- [ ] Collaborate (have a discussion)
- [ ] Get approval to merge 👍 
- [ ] Merge

### PR DESCRIPTION
**Pull request by smurffy**
closes #24

### CODE REVIEW

File: smurffy-baltmore.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Baltimore, MD
+
+### Restaurants
+Nicks Fish House
+Lexington Market
+
+### Places to Go
+Inner Harbor
+The National Aquarium
+American Visionary Art Museum
```
Comment (smurffy) [inline smurffy-baltmore.md:10]: great museum

File: smurffy-baltmore.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+### Current Temperature
+72 Degrees
```
Comment (smurffy) [inline smurffy-baltmore.md:14]: wow, nice temperature

**Review by smurffy — COMMENTED** [review]
review complete

**Comment by smurffy** [comment]
general speaking goes here 💃

**Review by alanrhannah — APPROVED** [review]
This looks great! :+1:

**Pull request merged**